app/src/androidTest/java/com/github/sdp_begreen/begreen/fragments/UserFragmentTest.kt

```diff
@@ -60,6 +60,7 @@ class UserFragmentTest {
             assertNotNull(recyclerView)
             assertTrue(recyclerView is RecyclerView)
         }
+        scenario.close()
```

Comment:
Could you use `@After` or something like that instead of inserting this in every test?